tests/src/JIT/Performance/CodeQuality/BenchF/Adams/Adams.cs

```diff
@@ -3,37 +3,41 @@
 // See the LICENSE file in the project root for more information.
 //
 // The Adams-Moulton Predictor Corrector Method adapted from Conte and de Boor
+// original source: adams_d.c
 
-using Microsoft.Xunit.Performance;
 using System;
 using System.Runtime.CompilerServices;
+using System.Diagnostics;
+#if XUNIT_PERF
 using Xunit;
+using Microsoft.Xunit.Performance;
+#endif
 
+#if XUNIT_PERF
 [assembly: OptimizeForBenchmarks]
 [assembly: MeasureInstructionsRetired]
+#endif
 
 public static class Adams
 {
 #if DEBUG
-    public const int Iterations = 1;
+    public static int Iterations = 1;
 #else
-    public const int Iterations = 200000;
+    public static int Iterations = 10000000;
```

Comment:
Please don't change these values. If you need the command line version to run more iterations, fix it in main.